### PR TITLE
iso-codes: use https urls, add head

### PR DESCRIPTION
**Pull request by vszakats**

### CODE REVIEW

**Comment by vszakats** [comment]
Strange, on the bot:
```
brew fetch --retry --build-bottle --force iso-codes			 FAILED
==> Downloading https://pkg-isocodes.alioth.debian.org/downloads/iso-codes-3.58.tar.xz
==> Retrying download
==> Downloading https://pkg-isocodes.alioth.debian.org/downloads/iso-codes-3.58.tar.xz
Error: Failed to download resource "iso-codes"
Download failed: https://pkg-isocodes.alioth.debian.org/downloads/iso-codes-3.58.tar.xz
```

While on desktop:
```
$ curl --version
curl 7.37.1 (x86_64-apple-darwin14.0) libcurl/7.37.1 SecureTransport zlib/1.2.5
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtsp smtp smtps telnet tftp 
Features: AsynchDNS GSS-Negotiate IPv6 Largefile NTLM NTLM_WB SSL libz 
$ curl -O https://pkg-isocodes.alioth.debian.org/downloads/iso-codes-3.58.tar.xz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 3688k  100 3688k    0     0  1293k      0  0:00:02  0:00:02 --:--:-- 1293k
```

**Comment by vszakats** [comment]
Download URL should be valid according to download page, too:
https://pkg-isocodes.alioth.debian.org/downloads/

**Comment by mikemcquaid** [comment]
Retrying on the bot. Perhaps a geolocation issue.

**Comment by vszakats** [comment]
Thanks! It went into green this time.

File: Library/Formula/iso-codes.rb [hunk 1]
```diff
@@ -1,9 +1,11 @@
 class IsoCodes < Formula
   desc "ISO language, territory, currency, script codes, and their translations"
-  homepage "http://pkg-isocodes.alioth.debian.org/"
-  url "http://pkg-isocodes.alioth.debian.org/downloads/iso-codes-3.58.tar.xz"
+  homepage "https://pkg-isocodes.alioth.debian.org/"
+  url "https://pkg-isocodes.alioth.debian.org/downloads/iso-codes-3.58.tar.xz"
   sha256 "86af5735dce6e4eff2b983e5d8aa9a3dea1b8db702333ff20be89e45f7f35a72"
 
+  head "git://git.debian.org/git/iso-codes/iso-codes.git"
```
Comment (DomT4) [inline Library/Formula/iso-codes.rb]: Do we need to regenerate the `configure` script here? Upstream ship a `configure` script but not sure how regularly it's updated.

Comment (vszakats) [inline Library/Formula/iso-codes.rb]: Haven't actually tested it, but looking at the repo source and comparing it with the distributed source package, all the same final makefiles are present in the repo, too. There is a script (`autogen.sh`) in the repo to regenerate them.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/3ac1b877b43268f2e038c418ac5a2fb7e33cb419. Thanks @vszakats!

**Pull request closed without merging**